docs/USAGE.rst

```diff
@@ -0,0 +1,366 @@
+
+Imported Files and Folders
+--------------------------
+
+Files are automatically copied from the local import directory to the ``AvailableToProcess`` folder in the ``WSI DeID`` collection in the WSI DeID. Files can have any folder structure; the folder structure is not significant in the import process. Excel files (identified by ending in .xls or .xlsx) and image files (anything else except for ignored files) will be imported. To facilitate bulk uploads, we ignore files ending in .txt, .xml, .zip from the import process -- this list can be easily changed.
+
+Import Process
+--------------
+
+From the ``AvailableToProcess`` folder (or any sub folder) in the WSI DeID, click on the ``Import`` button, as shown in the below screenshot.
+
+.. image:: screenshots/import_button_highlighted.png
+   :height: 100
+   :width: 200
+   :alt: import button
+
+A background process starts that scans through the mounted import directory, and does the following:
+
+- Each Excel file is parsed for a header row that has TokenID, ImageID, and ScannedFileName.
+- If there are any Excel files that do not have a header row, an error is generated and appears on the screen, and files are not imported.
+- If the same ScannedFileName is listed in multiple Excel files, the newest file is used by preference.
+- The ScannedFileName is expected to be just the file name (e.g., no folder path).
+
+After the image names and information in the metadata file are reconciled, the WSI DeID will classify images as one of the following:
+
+- ``present``: The image is listed in an Excel file and is already in the WSI DeID based on file path and matching file size. No action is performed.
+- ``added``: The image is listed in an Excel file and is not in the WSI DeID. It is added in the ``AvailableToProcess`` directory in a folder named TokenID with a filename ImageID.<extension>.
+- ``replaced``: The image is listed in an Excel file, is in the WSI DeID, but has a different file size from the image in the WSI DeID. The existing file is removed from the WSI DeID and re-added.
+- ``missing``: The image is listed in an Excel file but is not in the import directory. No action is performed.
+- ``unlisted``: The image is not listed in an Excel file but is in the import directory. No action is performed.
+- ``failed``: The listed file cannot be read as an image file.
+
+After all images and Excel metadata files have been processed, a message is displayed summarizing what images were in each of the five classifications above (e.g., "Import completed. 3 images added. 1 DeID Upload Excel file parsed. See the Excel file report for more details.). If you click on the "See the Excel report for more details" link, it will download an import report.
+
+Below is a screenshot of a message presented to the user after an import.
+
+.. image:: screenshots/import_message_highlighted.png
+   :height: 100
+   :width: 200
+   :alt: import message
+
```

Comment:
Add something to the effect of "see the error list table" for all possible results of performing the Import action.